### ISSUE TITLE
[Question] - Multi-user capabilities

### ISSUE THREAD

**Issue by kacer2k** [codingchili/kibana-mithril#48]
I'm thinking about start develop of to filter indices.
Did you think about best develop? It's for try something very fast.

Thanks :)

**Comment by codingchili** [comment]
Hello @kacer2k 

For multi-user I have tried to override the existing routes that deals with indices on the HAPI router, this didn't work. I remember having issues with modifying routes that were already added. At least it wasn't part of their public API, but this was on a much older version of HAPI that was used in Kibana 6.3.

I made another attempt where I started a new HAPI server that would proxy requests to Kibanas HAPI instance. It was brittle, as it depended on manually parsing api calls and determining the index that was requested. 

In the end I was not able to come up with a maintainable and secure solution that I was satisfied with. I am certain it can be done however, preferably with more insight on how the Kibana api works with index access/querying.
